components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/dao/OAuthConsumerDAOTest.java

```diff
@@ -161,12 +175,118 @@ public void testGetRequestToken() throws Exception {
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection3 = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection3);
+
+            OAuthConsumerDAO consumerDAO = new OAuthConsumerDAO();
+            consumerDAO.createOAuthRequestToken(CLIENT_ID, ACC_TOKEN, SECRET, CALLBACK, SCOPE);
+
+            try {
+                statement1 = connection3.prepareStatement(GET_ID);
+                statement1.setString(1, CLIENT_ID);
+                resultSet1 = statement1.executeQuery();
+                if(resultSet1.next()) {
+                     ID = resultSet1.getInt(1);
+                }
+
+                statement2 = connection3.prepareStatement(REQ_ID);
+                statement2.setInt(1, ID);
+                resultSet2 = statement2.executeQuery();
+                if(resultSet2.next()){
+                    req_Token = resultSet2.getString(1);
+                }
+            } finally {
+                IdentityDatabaseUtil.closeAllConnections(connection3, resultSet1, statement1);
+            }
+
+            assertEquals(REQ_TOKEN, REQ_TOKEN, "Checking whether the passed req_Token is set to the " +
+                    "REQ_TOKEN.");
+        }
+    }
+
+    @Test
+    public void testCreateOAuthRequestToken_02() throws Exception {
+
+        String callback_URL = null;
+        String c_Id = "fakeClientId";
+        String ac_Token = "fakeAccToken";
+        String secretFake = "fakeSecret";
+        String fakeScope = "fakeScope";
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection3 = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection3);
+
+            OAuthConsumerDAO consumerDAO = new OAuthConsumerDAO();
+            consumerDAO.createOAuthRequestToken(c_Id, ac_Token, secretFake, callback_URL, fakeScope);
+        }
+    }
+
+    @Test
+    public void testAuthorizeOAuthToken() throws Exception {
 
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        whenNew(Parameters.class).withNoArguments().thenReturn(mockedParameters);
```

Comment:
Can you debug and see whether at runtime the mockedParameters object returned when creating new instances of Parameter class. AFAIK the Parameter class should be added to PrepareForTest.

Reply:
My mistake. It was the class where the object is created. Not the object class. That was already done here.